code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -174,4 +174,11 @@
 	required_reagents = list("formaldehyde" = 1, "sodium" = 1, "chlorine" = 1, "lithium" = 1)
 	result_amount = 4
 	mix_message = "The mixture thins and loses all color."
-	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
\ No newline at end of file
+	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/chemical_reaction/rotatium
+	name = "Rotatium"
+	id = "Rotatium"
+	result = "rotatium"
+	required_reagents = list("lsd" = 1, "teslium" = 1, "fentanyl" = 1)
```

Comment:
we don't have fentanyl; I'd change this to meth, morphine, or hydrocodone, instead.

Reply:
Ugh, why do we still have reagent strings?